decentraland/unity-renderer
# feat: [SUB-BRANCH] Receive a friend request for improved friend requests

## How to test the changes?
1. Go to: https://play.decentraland.zone/?renderer-branch=feat/receive-a-friend-request-for-improved-friend-requests
2. Open the friends window.
3. Open the requests tab.
4. Press [R] key.
5. Notice a new fake friend request has been received.
6. Notice a new notification appears.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
